x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/api/ml_models/detach_ml_inference_pipeline.ts

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
I think we might want this and the delete logic in the pipelines folder instead of the ml_models` folder.